.travis.yml

```diff
@@ -4,7 +4,15 @@ rust:
   - stable
   - beta
   - nightly
-cache: cargo
+cache:
+  directories:
+    - $HOME/.cargo
+    - $HOME/.rustup
+before_cache:
+  - rm -rf /home/travis/.cargo/registry
+  - find ./target/debug -maxdepth 1 -type f -delete
```

Comment:
We don't cache `target` therefore we don't need to delete there something right?

Reply:
Ah, indeed, so we don't need a2d36107e770b07fbf164174727535a00f113871?

Reply:
I think we don't need it.

Reply:
Okay, dropped the commit.